server/notifications_events.js

```diff
@@ -274,3 +254,31 @@ function slimDownAllAcctPlansM(all_acct_plans_m) {
     return slim_acct_plan;
   });
 }
+
+function getEventId(event) {
+  if (event.request && event.request.transaction_id) {
+    return event.request.transaction_id;
+  }
+  if (event.eventPayload
+      && event.eventPayload.request
+      && event.eventPayload.request.transaction_id
+  ) {
+    return event.eventPayload.request.transaction_id
+  }
+  if (event.eventIdent && event.eventIdent.event_guid) {
+    return event.eventIdent.event_guid;
+  }
+  if (event.userAuditLogRequestDetails) {
+    const { requestActionType, requestID } = event.userAuditLogRequestDetails;
+    return `${requestActionType}_${requestID.replace(/^(req_id_\\s_)|^(req_id_)/, '')}`;
```

Comment:
Can't we just do: 

```suggestion
    return requestID.replace('req_id', requestActionType);
```

?

Reply:
Or:

```suggestion
    return `${requestActionType}_${requestID.replace('req_id_', ''}.trim()`;
```

Reply:
OK, we can't - discussed on slack

Reply:
`return requestID.replace(/^req_id_(\\s_)?/, requestActionType);`